ui/admin/app/components/form/host-catalog/aws/index.hbs

```diff
@@ -71,26 +72,28 @@
       </:body>
     </form.fieldset>
 
+    <Rose::Separator />
+
     <form.fieldset>
       <:title>{{t 'resources.host-catalog.form.provider-details'}}</:title>
       <:body>
         <form.radioGroup
-          @name='aws-plugins'
+          @name='aws_plugin'
           @selectedValue={{@model.compositeType}}
           @changed={{route-action 'changeType'}}
```

Comment:
The component itself should not trigger route actions.  Generally components should not be "aware" of the anything beyond the arguments passed into them.  Try passing in the route action via the route template.